www-nginx/ngx-brotli/files/ngx-brotli-20231109-use-system-brotli.patch

```diff
@@ -0,0 +1,103 @@
+From 0b42e57bb6fa4e039585f1ebac8584b8d45b7a2a Mon Sep 17 00:00:00 2001
```

Comment:
Please include a link to the Gentoo bug + the upstream PR at the top of the patch.